src/index.html

```diff
@@ -52,7 +52,12 @@ <h4>Dessert</h4>
             </ul>
         </div>
     </section>
-    <section class="popup" ></section>
+    <button class="comment">Comment</button>
```

Comment:
Can you please tell me the need for these buttons? I think it is for testing purposes but now we don't need it